### PR TITLE
Updated message description and comments

### PR DESCRIPTION
**Pull request by kothariaman**
### What does this Pull Request accomplish?
Updated comments and message description post feedback given by team after demo.

### Why should this Pull Request be merged?
Comments changes for better meaningful information.

### What testing has been done?
Locally tested the python version.

### CODE REVIEW

File: examples/nixnet/flexray-signal-single-point-input.py [hunk 1]
```diff
@@ -100,8 +100,9 @@ def check_for_error(status):
     )
     check_for_error(set_property_response.status)
 
+    # To get more meaningful data use this with output example
```
Comment (harsha-bhushan) [inline examples/nixnet/flexray-signal-single-point-input.py:103]: Avoid using 'meaningful'. Rephrase this to:
If no values are being written on the FlexRay port, the signals read would contain the default value of 0.0

File: examples/nixnet/flexray-signal-single-point-input.py [hunk 2]
```diff
@@ -100,8 +100,9 @@ def check_for_error(status):
...
     while i < 10:
-        # Update the signal data
+        # Update the signal data, the first read normally return the default payload values.
```
Comment (harsha-bhushan) [inline examples/nixnet/flexray-signal-single-point-input.py:105]: Revert this. I think the comment on 103 (rephrased statement) would convey this too.

File: examples/nixnet/can-signal-single-point-output.py [hunk 3]
```diff
@@ -87,9 +87,10 @@ def check_for_error(status):
     check_for_error(create_session_response.status)
 
     session = create_session_response.session
-
     print("Session Created Successfully.\n")
 
+    print("Writing 10 values to CAN Port.\n")
```
Comment (doshirohan) [inline examples/nixnet/can-signal-single-point-output.py:92]: I think this should be `CAN interface`

File: examples/nixnet/can-signal-single-point-output.py [hunk 4]
```diff
@@ -87,9 +87,10 @@ def check_for_error(status):
     check_for_error(create_session_response.status)
```
Comment (doshirohan) [inline examples/nixnet/can-signal-single-point-output.py:87]: Unused variable `SUCCESS` on line 52 above. GitHub doesn't let me add comment there, so adding it here.

File: examples/nixnet/ethernet-frame-stream-input-reader.py [hunk 5]
```diff
@@ -36,8 +36,7 @@
 
 CHOOSE_MONITOR_OR_ENDPOINT_TEXT = (
     "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
-    " to monitor all network traffic else just the enter key to use the endpoint path which filters"
-    " traffic based on VLAN ID and priority. :"
+    " to monitor all network traffic. :"
```
Comment (doshirohan) [inline examples/nixnet/ethernet-frame-stream-input-reader.py:39]: You previous message was fine, it just needed rewording for clarity.

> "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
    " to monitor all network traffic else press enter key to use the endpoint path which filters"
    " traffic based on VLAN ID and priority. :"
    
The change is in `press enter key`. I assume just pressing `enter` would configure it in filtered mode.


> Press 'm' followed by enter

Is this the behavior? Do you user need to press just `m` or `m + enter` to configure as monitor.

File: examples/nixnet/lin-signal-single-point-output.py [hunk 6]
```diff
@@ -114,7 +114,7 @@ def check_for_error(status):
         )
         check_for_error(write_state_response.status)
 
-    print("Values are incremented from 0 to 10 in this example.\n")
+    print("Writing 10 values to LIN Port.\n")
```
Comment (doshirohan) [inline examples/nixnet/lin-signal-single-point-output.py:117]: This should be `LIN interface`

**Review by doshirohan — APPROVED** [review]

File: examples/nixnet/ethernet-frame-stream-input-reader.py (hunk 5 above)
Comment (kothariaman) [reply examples/nixnet/ethernet-frame-stream-input-reader.py:39]: We need to press 'm' followed by enter because otherwise reading just a character will require importing getch and that anyways we ain't doing.

Comment (doshirohan) [reply examples/nixnet/ethernet-frame-stream-input-reader.py:39]: Sure, that makes sense. And for non-monitor mode just `enter` works, right?

Comment (kothariaman) [reply examples/nixnet/ethernet-frame-stream-input-reader.py:39]: No, for the else case it doesn't work with just `enter` and that's why i changed the comment.

Comment (doshirohan) [reply examples/nixnet/ethernet-frame-stream-input-reader.py:39]: Okay. In that case the second half of comment should read
> else press any other key followed by enter to use the endpoint path which filters

File: examples/nixnet/lin-signal-single-point-output.py (hunk 6 above)
Comment (harsha-bhushan) [reply examples/nixnet/lin-signal-single-point-output.py:117]: yes. interface is the 'word' to use in all examples.

File: examples/nixnet/flexray-signal-single-point-input.py (hunk 1 above)
Comment (kothariaman) [reply examples/nixnet/flexray-signal-single-point-input.py]: Done

File: examples/nixnet/flexray-signal-single-point-input.py (hunk 2 above)
Comment (kothariaman) [reply examples/nixnet/flexray-signal-single-point-input.py]: Done

File: examples/nixnet/can-signal-single-point-output.py (hunk 4 above)
Comment (kothariaman) [reply examples/nixnet/can-signal-single-point-output.py:86]: Done

File: examples/nixnet/can-signal-single-point-output.py (hunk 3 above)
Comment (kothariaman) [reply examples/nixnet/can-signal-single-point-output.py]: Done

File: examples/nixnet/lin-signal-single-point-output.py (hunk 6 above)
Comment (kothariaman) [reply examples/nixnet/lin-signal-single-point-output.py]: Done.

**Review by mgumble — APPROVED** [review]
